### ISSUE TITLE
Comparaison Mode

### ISSUE THREAD

**Issue by phlexib** [videovillage/Screen-Issues#46]
It would be awesome to be able to load two files  in a compare mode, with a few options for difference, stack, side by side...

**Comment by link266** [comment]
YES!!!!  Would be fantastic!

**Comment by mwjburton** [comment]
This would be so useful. I currently do a lot of comparisons by toggling between two open Screen windows, but a dedicated mode for this with extra options would be so useful.

**Comment by davidhlawrence** [comment]
Agreed! I toggle between two playlist items, but a comparison view would be much better.
